theme/static/js/resourceLandingAjax.js

```diff
@@ -1011,33 +1109,52 @@ function initializeDatePickers(){
 }
 
 // act on spatial coverage type change
-function setFileTypeSpatialCoverageFormFields(){
-    $("#div_id_type_filetype input:radio").change(function () {
-        if ($(this).val() == "point" && $(this).attr("checked") == "checked") {
-            $("#div_id_north_filetype").show();
-            $("#div_id_east_filetype").show();
-            $("#div_id_elevation_filetype").show();
-            $("#div_id_northlimit_filetype").hide();
-            $("#div_id_eastlimit_filetype").hide();
-            $("#div_id_southlimit_filetype").hide();
-            $("#div_id_westlimit_filetype").hide();
-            $("#div_id_uplimit_filetype").hide();
-            $("#div_id_downlimit_filetype").hide();
+function setFileTypeSpatialCoverageFormFields(logical_type){
```

Comment:
@alvacouch I’ll take the blame for this decision.  

The rationale is that I wanted a single point of truth.  I do not want xml metadata to be inconsistent with contents in the netCDF file.  I also want as much as possible of the metadata to be in the netCDF file, because it is tightly bound and hard to detach from the data (a big advantage of netCDF) whereas xml files and Django model metadata can easily be decoupled from the file and lost.  Also when the netCDF file is published through Hyrax only what is in the netCDF file is accessible.  Hyrax does not read HydroShare xml files.

When we let a person edit a resource, we have to let them edit the resource. We have to trust that if they change something it is for a reason.  We should hope that most of the writing of metadata to the netCDF will be adding metadata, rather than editing metadata, but there will be editing, and we hope that users take care to fix errors not introduce them.  

I do not think we can sustain a full git like trail of changes on all metadata.  If there is an RDF best practice that could record changes, I would be supportive of it being embedded in the resource data model for all resources.  I would support an enhancement feature that adds this across the system, but until we can do this, I think that the editing we have here is a realistic balanced and reasonably simple solution.

@gantian127 please note the points above.  This PR discussion becomes deeply hidden once the PR is merged, and some of these points should be made in the paper being written, as well as considered for future enhancements.